myVenv/Scripts/activate

```diff
@@ -0,0 +1,66 @@
+# This file must be used with "source bin/activate" *from bash*
```

Comment:
We don't want these files checked in - they'll vary on different operating systems and different people's machines. You can remove them now with `git rm --cached myVenv -r` & if you're going with this custom venv folder name then it'll be worth adding it to your `.gitignore` file